raiden/raiden_service.py

```diff
@@ -274,6 +277,12 @@ def __init__(self, chain, default_registry, private_key_bin, transport, discover
 
     def start(self):
         """ Start the node. """
+        # XXX Should this really be here? Or will start() never be called again
+        # after stop() in the lifetime of Raiden apart from the tests? This is
+        # at least at the moment prompted by tests/integration/test_transer.py
```

Comment:
I would say yes. It's true that we are only calling `start` twice from the tests, but were else would this go?

Reply:
the only place I would think this going would be the one test that failed when I did not have this here.

```
 raiden/tests/integration/test_transfer.py::test_direct_transfer_to_offline_node
```

It's the only test where we stop and restart a raiden service manually (at the moment)